File: bandeira/bandeira.go
package main

import (
	"fmt"
	"os"
	"strings"
)

const indicadorA = 0x1F1E6 // REGIONAL INDICATOR SYMBOL LETTER A

func main() {
	if len(os.Args) != 2 || len(os.Args[1]) != 2 {
		fmt.Println("Informe o código do país. Ex: BR ou br")
		os.Exit(0)
	}
	codigos := []string{}
	for _, c := range strings.ToUpper(os.Args[1]) {
		indicador := rune(c - 'A' + indicadorA)
		codigos = append(codigos, string(indicador))
	}
	// terminal do Ubuntu 18.04 ainda não reconhece REGIONAL INDICATORs..
	fmt.Println(strings.Join(codigos, ""))
}

File: bandeira/bandeira.py
#!/usr/bin/env python3

# https://tapoueh.org/blog/2017/07/playing-with-unicode/

import sys

INDICADOR_A = 0x1F1E6  # REGIONAL INDICATOR SYMBOL LETTER A

if len(sys.argv) != 2 or len(sys.argv[1]) != 2:
    print('Informe o código do país. Ex: BR ou br')
    sys.exit()

c1, c2 = [ord(c) - ord('A') + INDICADOR_A for c in sys.argv[1].upper()]

# terminal do Ubuntu 18.04 ainda não reconhece REGIONAL INDICATORs...
print(chr(c1) + chr(c2))
